src/ClientData/ScopeIdProvider.cpp

```diff
@@ -69,6 +74,13 @@ uint64_t NameEqualityScopeIdProvider::ProvideId(const TimerInfo& timer_info) {
   return orbit_client_data::kInvalidScopeId;
 }
 
+[[nodiscard]] std::vector<uint64_t> NameEqualityScopeIdProvider::GetAllProvidedScopeIds() const {
```

Comment:
So this doesn't return scope_ids corresponding to functions? This seems wrong as this class also provides ids for function. Also, is that correct in `LiveFunctionsDataView`?

Reply:
>So this doesn't return scope_ids corresponding to functions?

It does return all the provided ids, be it for hooked functions or not. So for now it's like `{1,2,3,4,5...}`

>So this doesn't return scope_ids corresponding to functions? This seems wrong as this class also provides ids for function. Also, is that correct in LiveFunctionsDataView?

In `LiveFunctionsDataView` I explicitly check if the id corresponds to a function or not.  Not sure what you mean, though.

Reply:
Oooh I forgot that `scope_id_to_name_` is also initialized by `Create`... Ignore.